src/components/Quote.vue

```diff
@@ -328,7 +339,7 @@ export default {
 		position: relative;
 		margin: auto;
 	}
-	&__file-preview {
+	&__image-preview {
 		line-height: 0.5 !important;
```

Comment:
```suggestion
		line-height: 1 !important;
```
For me it looks OK like this for all variations (file and/or caption, image and/or caption). Otherwise everything feels either cropped or too high
So I'm suggesting to drop all other changes :smile:

Reply:
it looks trimmed, no ? 
![image](https://github.com/nextcloud/spreed/assets/84044328/c7125fd1-7f72-49c9-a6cc-bdbb799cb547)

Reply:
I see a bit, thanks. Maybe 1.2?

Reply:
Still 
Before (line-height: 1.2) | Testing
--|--
![image](https://github.com/nextcloud/spreed/assets/84044328/e3508a2a-8696-4c71-bdcd-ffa0065b76dd)| ![image](https://github.com/nextcloud/spreed/assets/84044328/93dd6873-e0ea-4e50-b078-8fcd7e071480)|


I was thinking about keeping it to default ( 1.5) but caption looks really far IMO 
Before (line-height: 1.5) | After (line-height: 1)
--|--
![image](https://github.com/nextcloud/spreed/assets/84044328/80cb1b22-30f1-492b-8223-aa6c22520231)|![image](https://github.com/nextcloud/spreed/assets/84044328/3b72ee5b-4eb3-4abc-9c4d-5f81c6778f12)

so drop it or keep it ?

Reply:
Let's keep it, but using existed computed (isFileShare, isFileShareWithoutCaption)